go-controller/pkg/ovn/master.go

```diff
@@ -262,6 +262,13 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 		klog.Info("SCTP support detected in OVN")
 	}
 
+	// Create a cluster-wide port group that all logical switch ports are part of
+	oc.clusterPortGroupUUID, err = createPortGroup(clusterPortGroupName, clusterPortGroupName)
```

Comment:
we only need a cluster port group with the mp0 ports in it. That will be enough to get the ACLs on each switch. It looks like you are already doing that? So the ocmment on 265 is not accurate?

Reply:
This patch do add all the logical ports to the port-group. I didn't find any document that points out that adding management port will be enough. As per our email communication with ovn team, the idea was to add all the logical ports to the port group. I recently learned it from Ilya that it should work if you only add management port. I assume that it works, this will reduce the memory usage further. So will remove the code that adds ports to port-group. 
On a side note: Does that mean, if you add management port to the port-group, there is no point in adding any other logical port to that group, because it will anyways apply on the logical switch?

Reply:
right that's my understanding

Reply:
Please triple-confirm this understanding of ACLs first though...

Reply:
@dcbw Yes, i had a discussion over irc and following bugs are result of that discussion as well
https://bugzilla.redhat.com/show_bug.cgi?id=1885670
https://bugzilla.redhat.com/show_bug.cgi?id=1885668